pyproject.toml

```diff
@@ -1,6 +1,77 @@
+[project]
+name = "Flask-DebugToolbar"
+version = "0.15.1"
+description = "A toolbar overlay for debugging Flask applications."
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+author = [{ name = "Michael van Tellingen" }]
+maintainers = [{ name = "Pallets Ecosystem", email = "contact@palletsprojects.com" }]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Framework :: Flask",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "flask>=2.3.0",
+]
+
+[project.urls]
+Documentation = "https://flask-debugtoolbar.readthedocs.io"
+Changes = "https://github.com/pallets-eco/flask-debugtoolbar/releases/"
+Source = "https://github.com/pallets-eco/flask-debugtoolbar/"
+Chat = "https://discord.gg/pallets"
```

Comment:
I don't monitor this, I doubt any of the other current maintainers do... totally cool with leaving it in, just an FYI this redirects folks to something we don't really look at.

Reply:
For better or worse, we're building the Pallets community in one place. The place for chat is our official Discord server. If I could make a new decision today it would probably be Zulip or Matrix, but that's not something I have the bandwidth to work on right now. Regardless, it would be somewhere, and I would hope that members of the pallets-eco community would congregate there.

Additionally, while I know Flask-DebugToolbar is maintained fairly actively, that's actually unusual for a project in Pallets-Eco right now. The goal is to make it easier for members of the community to join in as maintainers to _any_ of the pallets-eco projects. Part of that is having a single place to go for coordination.

Reply:
Sounds great! 

> while I know Flask-DebugToolbar is maintained fairly actively, that's actually unusual for a project in Pallets-Eco right now.

Amazing, because we considered it a project almost on life support. 😄 

> The goal is to make it easier for members of the community to join in as maintainers to any of the pallets-eco projects. Part of that is having a single place to go for coordination.

👍 for reducing friction for community to get involved. 😍